docs/layers.md

```diff
@@ -242,6 +242,8 @@ Landuse polygons from OpenStreetMap representing parks, forests, residential, co
 
 Zooms 4 and 5, 6 and 7 includes a mix of Natural Earth `urban_area` (zooms 0-9 only) features and OpenStreetMap data for `national_park`, `protected_area`, and `nature_reserve` only. After that more more feature kinds are included, and they have a richer set of properties including `sport`, `religion`, `surface`, `attraction`, `zoo`, and `natural`. Feature selection is filtered per zoom until zoom 15.
 
+At mid-zooms, between 9-12, some landuse polygons are merged to reduce payload size. To facilitate this, the name of the landuse area may be dropped for small polygons. When polygons are merged, the original `id` properties are dropped, and the `area` is re-calculated for the new size.
```

Comment:
Generally this is true, but for a few kinds we show them as earlier as zoom 2 (https://github.com/tilezen/vector-datasource/blob/master/yaml/landuse.yaml#L5-L13).